assets/js/lib/api/wanda.js

```diff
@@ -13,4 +13,4 @@ export const getLastExecutionByGroupID = (groupID) =>
   wandaClient.get(`/api/checks/groups/${groupID}/executions/last`);
 
 export const getCatalog = (env) =>
-  wandaClient.get(`/api/checks/catalog`, { params: env })
+  wandaClient.get(`/api/checks/catalog`, { params: env });
```

Comment:
shouldn't the linter catch this?

Reply:
It was a formatting issue, for some reason previous formatting tasks didn't catch 🤷‍♂️

Reply:
@CDimonaco can we investigate? are we checking the formatting?

Reply:
Yes in the CI we have the correct step, I will investigate further.